system/jlib/jlog.cpp

```diff
@@ -2324,6 +2331,53 @@ MODULE_EXIT()
     thePassAllFilter = NULL;
 }
 
+void updateContainerizedLogMsgHandlerDetail(LogMsgDetail logDetail)
+{
+    updateContainerizedLogMsgHandler(getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, logDetail));
+}
+
+void updateContainerizedLogMsgHandlerLogFields(unsigned fields)
+{
+	theStderrHandler->setMessageFields(fields);
+}
+
+void updateContainerizedLogMsgHandlerFromConfig()
+{
+	if (queryComponentConfig().hasProp("logging"))
+	{
+		updateContainerizedLogMsgHandler(&queryComponentConfig());
+	}
+}
+
+void updateContainerizedLogMsgHandler(ILogMsgFilter * newFilter)
+{
+	if (newFilter != nullptr)
+	{
+		theManager->changeMonitorFilter(theStderrHandler, newFilter);
+	}
+}
+
+void updateContainerizedLogMsgHandler(IPropertyTree * tree)
+{
+	if (tree)
+	{
+		if (tree->hasProp("logging/logfields"))
```

Comment:
We probably want this as @logFields, to be consistent with our config files/yaml.
Other properties below too should be attributes (lower case yaml scalar values imply attributes), and camelCased.